spec/rspec/scenarios_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+require 'capybara/rspec'
+
+RSpec.configuration.before(:each, { file_path: "./spec/rspec/features_spec.rb" } ) do
```

Comment:
Nice catch! Actually `@in_filtered_hook = true` was needed for rbnx tests! They pass now :)